test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -563,11 +564,37 @@ TEST(mongoContextProvidersQueryRequest, noPatternAttrsAll)
   /* Check response is as expected */
   EXPECT_EQ(SccOk, ms);
 
-  EXPECT_EQ(SccFound, res.errorCode.code);
-  EXPECT_EQ("Found", res.errorCode.reasonPhrase);
-  EXPECT_EQ("http://cr1.com", res.errorCode.details);
+  EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
I never liked this SccNone.

First doubt, should any mongoBackend 'main' functions ever return with SccNone?
[ I understood this should never happen ... to be considered an error if it ever happened ]

If this SccNone is in errorCode.code, should it be treated as an OK?
And the calling layer will always have to test for ((code == SccOk) || (code == SccNone)).

[ In some cases the SccNone is transformed into a 500 - internal error ... ]

[[ I would initialize the errorCode to 200 OK and in case there is any error, that would be changed by mongBackend.
However, I think we've discussed this more times than the ciP :-D. ]]

I think we perhaps need to revisit this whole thing.